aks_rcs_client/aks_rcs_client.py

```diff
@@ -0,0 +1,71 @@
+#usr/bin/env python3
+import rclpy
+import paho.mqtt.client as mqtt
+from rclpy.node import Node
+from std_msgs.msg import String
+
+class aks_rcs_client(Node):
+    def __init__(self):
+        super().__init__("aks_rcs_client")
+
+        self.declare_parameters(
+            namespace='',
+            parameters=[
+                ('mqtt_host', "127.0.0.1"),                     # ADDRESS TO MQTT BROKER
+                ('mqtt_port', 1883),                            # PORT TO MQTT BROKER
+                ('team_id', "AMP"),                             # USED FOR CLIENT NAME AND MQTT KART STATE TOPIC
+                ('mqtt_track_state_topic', "track_state"),
+                ('mqtt_kart_state_topic', "kart_state/"),
+                ('ros_track_state_topic', "race_state"),
+                ('ros_kart_state_topic', "kart_state")          # TOPIC TO PUBLISH STATE DATA WITHIN ROS
+            ]
+        )
+
+        client_name = self.get_parameter('team_id').value
+        mqtt_host = self.get_parameter('mqtt_host').value
+        mqtt_port = self.get_parameter('mqtt_port').value
+        mqtt_track_topic = self.get_parameter('mqtt_track_state_topic').value
+        global mqtt_kart_topic ; str(self.get_parameter('mqtt_kart_state_topic').value) + str(self.get_parameter('team_id').value) # STRING NOT PARAMETER OBJECT LIKE OTHERS
```

Comment:
Instead of using `global`, set it to a class parameter with `self`. Also, semicolon looks like a typo.

Look above for other change.